src/wallet.js

```diff
@@ -1578,6 +1578,7 @@ const wallet = {
     storage.removeItem('wallet:lastGeneratedIndex');
     storage.removeItem('wallet:lastUsedIndex');
     storage.removeItem('wallet:lastUsedAddress');
+    storage.removeItem('wallet:multisig');
```

Comment:
Maybe this should be above inside the `if (cleanAccessData) {`, what do you think?

Reply:
Definetely, I will change it

Reply:
Done